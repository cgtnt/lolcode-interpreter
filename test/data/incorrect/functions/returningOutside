HAI

FOUND YR 5

KTHXBYE
