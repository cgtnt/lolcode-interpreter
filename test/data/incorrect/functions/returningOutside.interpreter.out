Line 3: Cannot FOUND YR or GTFO from main program block
